src/dev/src/app/timeline/angular/timeline-angular.html

```diff
@@ -0,0 +1,85 @@
+<!--
+  ~ Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+  ~ This software is released under MIT license.
+  ~ The full license information can be found in LICENSE in the root directory of this project.
+  -->
+
+<h3>Angular component</h3>
+
+<h4>Horizontal timeline</h4>
+
+<clr-timeline>
+  <clr-timeline-step clrState="not-started">
+    <clr-timeline-step-header>11:59 am</clr-timeline-step-header>
+    <clr-timeline-step-title>Add KMS</clr-timeline-step-title>
+    <clr-timeline-step-description>Root CA certificate requested.</clr-timeline-step-description>
+  </clr-timeline-step>
+  <clr-timeline-step clrState="current">
+    <clr-timeline-step-header>11:59 am</clr-timeline-step-header>
+    <clr-timeline-step-title>Add KMS</clr-timeline-step-title>
+    <clr-timeline-step-description>
+      Root CA certificate requested. Upload it to the KMS to complete the connection.
+      <button class="btn btn-sm">Action</button>
+    </clr-timeline-step-description>
+  </clr-timeline-step>
+  <clr-timeline-step class="gemini-ignore" clrState="processing">
+    <clr-timeline-step-header>11:59 am</clr-timeline-step-header>
+    <clr-timeline-step-title>Make vCenter trust KMS</clr-timeline-step-title>
+    <clr-timeline-step-description>
+      Root CA certificate requested. Upload it to the KMS to complete the connection. Third sentence is very long
+      and very long.
+    </clr-timeline-step-description>
+  </clr-timeline-step>
+  <clr-timeline-step clrState="success">
+    <clr-timeline-step-header>11:59 am</clr-timeline-step-header>
+    <clr-timeline-step-title>Make KMS trust vCenter</clr-timeline-step-title>
+    <clr-timeline-step-description>Upload it to the KMS to complete the connection. Third
+      sentence.
+      <button class="btn btn-sm btn-link">Action</button>
+    </clr-timeline-step-description>
+  </clr-timeline-step>
+  <clr-timeline-step clrState="error">
+    <clr-timeline-step-header>11:59 am</clr-timeline-step-header>
+    <clr-timeline-step-title>Connected</clr-timeline-step-title>
+    <clr-timeline-step-description>No. It's not connected.</clr-timeline-step-description>
+  </clr-timeline-step>
+</clr-timeline>
+
+<h4>Vertical timeline</h4>
+
+<clr-timeline clrLayout="vertical" style="width: 50%">
```

Comment:
The `clrLayout` input is typed to `TimelineLayout`. Taking a string for the input here seems odd to me.